src/Presenters/ApiPresenter.php

```diff
@@ -240,4 +243,37 @@ private function log($apiIdentifier, $authorization, $response)
             'userAgent' => $userAgent,
         ]), HermesMessage::PRIORITY_LOW);
     }
+
+    private function hasPathLogEnabled(string $path): bool
+    {
+        $apiLogPathsConfig = Strings::trim($this->applicationConfig->get('enabled_api_log_paths'));
```

Comment:
I more prefer "plug&play" configuration otherwise we have to re-deploy every-time we need to change it . If whitelist will be replaced with blacklist user can't break anything. 
But if you prefer configuration via config.neon i prepare it.

Reply:
Done